app/components/Sidebar/components/Version.tsx

```diff
@@ -9,18 +9,17 @@ export default function Version() {
 
   React.useEffect(() => {
     async function loadReleases() {
-      let out = 0;
       const res = await fetch(
         "https://api.github.com/repos/outline/outline/releases"
       );
       const releases = await res.json();
 
-      for (const release of releases) {
-        if (release.tag_name === `v${version}`) {
-          return setReleasesBehind(out);
-        } else {
-          out++;
-        }
+      if (Array.isArray(releases)) {
```

Comment:
This change should be removed/separate PR with explanation

Reply:
Explanation - I triggered the API rate-limiting for my IP. It was a pain when the request failed, and I had to close the error screen every time to get to the `features` page.